tests/wpt/mozilla/meta/MANIFEST.json

```diff
@@ -39229,6 +39229,12 @@
      {}
     ]
    ],
+   "mozilla/webgl/clear.html": [
```

Comment:
@emilio We can't put tests there without causing issues for syncing with Khronos, right?

Reply:
Yeah, think so.

Reply:
No - we already have test files in that directory that are not part of the sync. It's fine to add additional ones.